test/timer.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { GameStartType, SlippiGame } from "../src";
+import { frameToGameTimer } from "../src/utils/gameTimer";
+
+describe("when calculating the in-game timer", () => {
+  it("should calculate the timer correctly when increasing", () => {
+    const timer_increasing = new SlippiGame("./slp/timer_increasing.slp");
+    const timer_increasing_gameinfo = timer_increasing.getSettings();
+    expect(frameToGameTimer(2014, timer_increasing_gameinfo as GameStartType)).toBe("00:33.57");
```

Comment:
This test file is really only testing the `frameToGameTimer` conversion function. Because of that, you don't need to even declare `new SlippiGame()` or access `game.getSettings()`. Just pass in a mock `GameStartType` and delete the test SLPs.

You can also probably just co-locate this test with the `gameTimer.ts` file in the same `utils` folder but called `gameTimer.spec.ts`.